traitlets/traitlets.py

```diff
@@ -2269,95 +2291,110 @@ def validate(self, obj, value):
         self.error(obj, value)
 
 
-class Container(Instance):
-    """An instance of a container (list, set, etc.)
-
-    To be subclassed by overriding klass.
-    """
-    klass = None
-    _cast_types = ()
-    _valid_defaults = SequenceTypes
-    _trait = None
+class Mutable(TypeCast):
 
-    def __init__(self, trait=None, default_value=None, **kwargs):
-        """Create a container trait type from a list, set, or tuple.
-
-        The default value is created by doing ``List(default_value)``,
-        which creates a copy of the ``default_value``.
-
-        ``trait`` can be specified, which restricts the type of elements
-        in the container to that TraitType.
-
-        If only one arg is given and it is not a Trait, it is taken as
-        ``default_value``:
-
-        ``c = List([1, 2, 3])``
-
-        Parameters
-        ----------
+    def __init__(self, *args, **kwargs):
+        super(Mutable, self).__init__(*args, **kwargs)
+        self._default_progenitor = self.default_value
+        self.default_value = Undefined
 
-        trait : TraitType [ optional ]
-            the type for restricting the contents of the Container.  If unspecified,
-            types are not checked.
+    def make_dynamic_default(self):
+        return copy.copy(self._default_progenitor)
```

Comment:
Shouldn't this be `copy.copy(self.default_value)`? Why is the default_value unused and a separate private attribute used for the same purpose?